mllib-dal/src/main/native/PCADALImpl.cpp

```diff
@@ -35,131 +35,130 @@ typedef double algorithmFPType; /* Algorithm floating-point type */
 static void doPCADALCompute(JNIEnv *env, jobject obj, int rankId,
                             ccl::communicator &comm, NumericTablePtr &pData,
                             int nBlocks, jobject resultObj) {
+    using daal::byte;
     auto t1 = std::chrono::high_resolution_clock::now();
 
-    pca::Distributed<step1Local, algorithmFPType, pca::svdDense> localAlgorithm;
+    const bool isRoot = (rankId == ccl_root);
+
+    covariance::Distributed<step1Local, algorithmFPType> localAlgorithm;
 
     /* Set the input data set to the algorithm */
-    localAlgorithm.input.set(pca::data, pData);
+    localAlgorithm.input.set(covariance::data, pData);
 
-    /* Compute PCA decomposition */
+    /* Compute covariance */
     localAlgorithm.compute();
 
     auto t2 = std::chrono::high_resolution_clock::now();
     auto duration =
-        std::chrono::duration_cast<std::chrono::seconds>(t2 - t1).count();
-    std::cout << "PCA (native): local step took " << duration << " secs"
-              << std::endl;
+    std::chrono::duration_cast<std::chrono::seconds>(t2 - t1).count();
+    std::cout << "Covariance (native): local step took " << duration << " secs"
```

Comment:
```suggestion
    std::cout << "PCA (native): Covariance local step took " << duration << " secs"
```

Reply:
It's still a PCA (native) step, you can put covariance after colon. Pls also check below.